src/main/resources/avro/readmethods.avdl

```diff
@@ -14,8 +14,10 @@ to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
 */
 record SearchReadsRequest {
-  /** If nonempty, restrict this query to reads within the given readgroups. */
-  array<string> readGroupIds = [];
+  /**
+  The ReadGroup to search.
+  */
+  string readGroupId;
```

Comment:
Adding `readGroupsetId` addresses (1) only. Reverting this change or adding `array<string> readGroupSetIds` addresses both (1) and (2).

We've had this feature requested in the past, but others can probably provide better commentary on the use cases and how applicable it is today.